.gitignore

```diff
@@ -24,3 +24,5 @@ reports/
 
 build/
 /tmp/
+
+!.golangci.yaml
```

Comment:
IMHO you have issues because of line 8, IMHO we should remove lines 8 and 26 as they don't make sense, WDYT @VeereshAradhya ?

Reply:
I think I only need to add !.golangci.yaml to the local gitignore and not commit this change.

Reply:
I don't understand why you would need to add it to gitignore, even locally